parquet-thrift/src/main/java/parquet/thrift/ThriftSchemaConverter.java

```diff
@@ -93,7 +94,7 @@ private StructType toStructType(TStructDescriptor struct) {
                         Requirement.fromType(field.getFieldMetaData().requirementType);
         children.add(toThriftField(field.getName(), field, req));
       }
-      return new StructType(children);
+      return new StructType(children, TBaseUtil.isUnion(struct.getThriftClass()));
```

Comment:
I don't see why we need a TBaseUtil at all and don't declare "isUnion" private in this class.
I usually try to avoid "Util" classes. It could always be moved if some other class needs it.